### PR TITLE
FALCON-1804.Non-SLA feed throws NullPointerException

### PR DESCRIPTION
**Pull request by sandeepSamudrala**

### CODE REVIEW

File: rerun/src/main/java/org/apache/falcon/rerun/handler/LateRerunHandler.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public void handleRerun(String cluster, String entityType, String entityName, St
         try {
             Entity entity = EntityUtil.getEntity(entityType, entityName);
             int intRunId = Integer.parseInt(runId);
-            Date msgInsertTime = EntityUtil.parseDateUTC(nominalTime);
+            Date msgInsertTime = new Date(msgReceivedTime);
```
Comment (pallavi-rao) [inline rerun/src/main/java/org/apache/falcon/rerun/handler/LateRerunHandler.java]: Unrelated change. Please remove.

Comment (sandeepSamudrala) [inline rerun/src/main/java/org/apache/falcon/rerun/handler/LateRerunHandler.java]: Thanks @pallavi-rao for pointing out.
Closed this pull request and raised another pull request with only a single commit.
